packages/api/lib/testUtils.js

```diff
@@ -50,6 +51,7 @@ function fakeFileFactory(params = {}) {
     bucket: randomString(),
     fileName,
     key: fileName,
+    fileType: 'data', // Default to data, params should override
```

Comment:
This should be removed, as granule files with fileTypes can be added as a parameter.